pipeline/04-interpret.R

```diff
@@ -152,6 +152,11 @@ if (comp_enable) {
   training_data <- read_parquet(paths$input$training$local) %>%
     filter(!ind_pin_is_multicard, !sv_is_outlier) %>%
     as_tibble()
+  training_data_prepped <- recipes::bake(
+    object = lgbm_final_full_recipe,
+    new_data = training_data,
+    all_predictors()
+  )
```

Comment:
This isn't new, but rather has been moved up from a later code block since it's now used in the `extract_tree_weights` call as well.